src/bindings/js/node/tests/tensor.test.js

```diff
@@ -1,113 +1,113 @@
 var ov = require('../build/Release/ov_node_addon.node');
+const assert = require("assert");
+const {test, describe, it} = require("node:test");
```

Comment:
Looks like eslint doesn't check formatting 
```suggestion
const assert = require('assert');
const { test, describe, it } = require('node:test');
```